pyasdf/asdftypes.py

```diff
@@ -108,13 +109,37 @@ def from_yaml_tag(self, tag):
 
     @lru_cache(5)
     def has_hook(self, hook_name):
+        """
+        Returns `True` if the given hook name exists on any of the managed
+        types.
+        """
         for cls in self._all_types:
             if hasattr(cls, hook_name):
                 return True
         return False
```

Comment:
They should probably be documented in one place somewhere, but aren't currently.